### ISSUE TITLE
Firefox extension is inaccessible with NVDA, and no obvious way of activating the Chrome extension with NVDA

### ISSUE THREAD

**Issue by Neurrone** [dequelabs/axe-core#1170]
Hi,

With firefox 62, the extension is completely inaccessible to NVDA 2018.3. After activating the AXE tab in Firefox's developer tools, attempting to tab into the Axe window and arrow around results in no spoken feedback at all. 

I tried using the Chrome extension as well, but could not find the Axe tab in Chrome's development tools. The closest was the accessibility tab, which is Chrome's built-in support for displaying the accessibility tree.

Hope this gets fixed soon, especially because this is happening to an extension that checks for accessibility.

**Comment by WilcoFiers** [comment]
Thanks for reporting. We're aware of the Chrome keyboard problem. We filed a bug with the Chrome team when the problem was first introduced. It's a rather unfortunate one, as there doesn't seem to be a workaround for it. You can see it here: https://bugs.chromium.org/p/chromium/issues/detail?id=872438 That bug seems to have been fixed, and it looks like they're planning to ship the fix in Chrome version 72. 

As for Firefox, I'm not able to reproduce the issue you're having. I've tried this with NVDA 2018.3.2, with both Firefox 61 and Firefox 62. I noticed that in the style editor, there seems to be a keyboard trap, which could get you stuck and unable to change to other devtools panels. Closing and reopening the devtools (`ctrl` + `shift` + `i`) gets you out of that. To switch between panels, use `ctrl` + `]`. The second problem with that is that the custom devtools panels, such as the axe panel don't get correctly announced. An internal URL seems to be announced before the actual name of the panel. It does get announced when you open it. Once open you can simply tab inside and use navigation as usual. That doesn't seem to have changed.

One thing of note is that we are working on a screen reader guide. Problems with accessibility of developer tools keep coming up so we're working to keep an up to date instructions where we can inform people of potential bugs and known work around.

I hope this helps. If you can provide more information on how to reproduce the Firefox issue, that would be helpful.

**Comment by Neurrone** [comment]
@WilcoFiers that's odd, I don't have the same trouble with Firefox when I tried today, so it is hopefully a one-off NVDA and/or firefox problem. I wasn't doing anything differently today. I'll re-open this issue if it happens again.

Ctrl+] also works for me in chrome, though I would have never known to do that because I can't find the AXE tab at all through normal means.

I've found some usability problems in the Firefox extension, which I will detail in a separate issue.

**Comment by Neurrone** [comment]
@WilcoFiers I'd also be interested in reading and contributing to the guide you mentioned. The state of accessibility for browser dev tools and IDEs more generally is still really bad.
